### PR TITLE
[KOGITO-2565] - Upgrade operator-sdk on dynamic slave image for kogit…

### PR DESCRIPTION
**Pull request by Kaitou786**
…o-cloud-operator PR check

See: https://issues.redhat.com/browse/KOGITO-2565

### CODE REVIEW

**Comment by Kaitou786** [comment]
Should be merged only after https://github.com/kiegroup/kogito-cloud-operator/pull/399

**Review by msmagnanijr — APPROVED** [review]

File: kogito-cloud-operator-jenkins-slave/image.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ modules:
     - name: org.kie.kogito.mercurial
       version: "5.3.1"
     - name: org.kie.kogito.operator-sdk
-      version: "v0.15.1"
+      version: "v0.18.2"
```
Comment (radtriste) [inline kogito-cloud-operator-jenkins-slave/image.yaml:14]: could you update as well the main image.yaml version to 0.12.0, as the change will impact this version ?

**Review by radtriste — CHANGES_REQUESTED** [review]

File: kogito-cloud-operator-jenkins-slave/image.yaml (hunk 1 above)
Comment (Kaitou786) [reply kogito-cloud-operator-jenkins-slave/image.yaml:14]: Updated the image version

**Review by radtriste — APPROVED** [review]

File: kogito-cloud-operator-jenkins-slave/modules/operator-sdk/v0.18.x/module.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: org.kie.kogito.operator-sdk
+version: "v0.18.2"
+description: Install operator-sdk from sources
+
+artifacts:
+  - name: operator-sdk-v0.18.2-x86_64-linux-gnu
+    url: https://github.com/operator-framework/operator-sdk/releases/download/v0.18.2/operator-sdk-v0.18.2-x86_64-linux-gnu
```
Comment (spolti) [inline kogito-cloud-operator-jenkins-slave/modules/operator-sdk/v0.18.x/module.yaml:7]: add the md5 hash here :)

**Review by spolti — CHANGES_REQUESTED** [review]

File: kogito-cloud-operator-jenkins-slave/modules/operator-sdk/v0.18.x/module.yaml (hunk 2 above)
Comment (Kaitou786) [reply kogito-cloud-operator-jenkins-slave/modules/operator-sdk/v0.18.x/module.yaml:7]: Added

**Review by spolti — APPROVED** [review]

**Pull request merged**